include/openmc/dagmc.h

```diff
@@ -184,6 +190,12 @@ class DAGUniverse : public Universe {
                              //!< generate new material IDs for the universe
   bool has_graveyard_; //!< Indicates if the DAGMC geometry has a "graveyard"
                        //!< volume
+  std::map<std::string, vector<std::string>>
+    instance_material_overrides; ///!< Map of material overrides
```

Comment:
More generally I think we can call this `material_overrides`. If an entry in this object is length 1, it can effectively an override for a cell without any additional instances.